### PR TITLE
Rework rabbitmq

### LINKED ISSUE #11: Rework RabbitMQ
A/C
- [ ] use stream type queues for things we need to keep around
- [ ] simplify setup script (make use of things in the base image)

### PR DESCRIPTION
**Pull request by jbygdell**
* Updates to latest version; Rabbitmq `3.11.11`, Erlang `25.3`
* Makes use of ENVs defined in the base image

Closes #11

### CODE REVIEW

File: rabbitmq/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+We use [RabbitMQ](https://hub.docker.com/_/rabbitmq) including the management plugins.
+
+## Configuration
+
+The following environment variables can be used to configure the broker:
+
+| Variable                 | Description                                                                               | Default value |
+| :----------------------- | :---------------------------------------------------------------------------------------- | :------------ |
+| `RABBITMQ_DEFAULT_USER`  | Default user (with admin rights)                                                          | `guest` |
+| `RABBITMQ_DEFAULT_USER`  | Password for the above user                                                               | `guest` |
```
Comment (norling) [inline rabbitmq/README.md:12]: Probably something like this?
```suggestion
| `RABBITMQ_DEFAULT_PASS`  | Password for the above user                                                               | `guest` |
```

**Review by norling — APPROVED** [review]

**Review by blankdots — DISMISSED** [review]
one questions other than that lgtm

File: rabbitmq/definitions.json [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        },
+        {
+            "name": "ingest",
+            "vhost": "sda",
+            "durable": true,
+            "auto_delete": false,
+            "arguments": {}
+        },
+        {
+            "name": "mappings",
```
[the remark this answers is not in the document]
Comment (jbygdell) [reply rabbitmq/definitions.json:87]: How long retention would we like for those messages, one week/month?

[the remark this answers is not in the document]
Comment (blankdots) [reply rabbitmq/definitions.json:87]: Month should be enough

**Review by blankdots — APPROVED** [review]

**Review by norling — APPROVED** [review]

**Pull request merged**